internal/host/validator.go

```diff
@@ -1172,8 +1172,9 @@ func (v *validator) printDefaultRoute(c *validationContext, status ValidationSta
 	}
 }
 
-func isNonePlatformMultiNodeInstallation(c *validationContext) bool {
-	return swag.BoolValue(c.cluster.UserManagedNetworking) && !common.IsSingleNodeCluster(c.cluster)
+func shouldValidateAPIDomainName(c *validationContext) bool {
```

Comment:
Name is a bit misleading since it's also called in `isAppsDomainNameResolvedCorrectly` which doesn't have anything to do with the (k8s) API

Should probably be something like `shouldValidateDnsResolution`

Reply:
But it's not just API, it's also for `isAppsDomainNameResolvedCorrectly`

Reply:
Oh, I see, `shouldValidateDnsResolution` is a good choice then